cmd/gpud/command/login.go

```diff
@@ -82,7 +82,7 @@ func cmdLogin(cliContext *cli.Context) error {
 		if err != nil {
 			hostname = "UnknownName"
 		}
-		if err := login.Login(hostname, token, endpoint, components, uid); err != nil {
+		if err := login.SendRequestToOldAPI(hostname, token, endpoint, components, uid); err != nil {
```

Comment:
We don't need it anymore

Reply:
@eahydra Removed.

And will implement the rest of the login workflow in the following PR